tests/foreman/ui/test_gpgkey.py

```diff
@@ -1,9 +1,9 @@
 # -*- encoding: utf-8 -*-
 """Test class for GPG Key UI"""
 
+import random
```

Comment:
@elyezer and I had a discussion about this in the past.  We decided to use this format.  Example: random.randint or random.choice, entities.Organization to add more readability in the code.  Also if I am not wrong, import random or random.choice will not give any performance improvement as the module is loaded to memory anyway.